tests/integration/test_logcollector/test_macos/test_macos_format_only_future_events.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+import fnmatch
+import os
+
+import pytest
+from wazuh_testing import logcollector
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.remote import check_agent_received_message
+from wazuh_testing.tools.services import control_service
+
+# Marks
+
+pytestmark = [pytest.mark.darwin, pytest.mark.tier(level=1)]
```

Comment:
We should add this one to the 0 tier IMO.

Reply:
Done 58f4bb663ce1da50f8b24241a5ba741cebcffe72